client/galaxy/scripts/components/Workflow/Invocations.vue

```diff
@@ -23,85 +21,71 @@
                 <template v-slot:row-details="row">
                     <b-card>
-                        <!-- set provideContext to false, since the table itself provides this information -->
-                        <workflow-invocation-state :invocation-id="row.item.id" :provide-context="false" />
+                        <workflow-invocation-state :invocation-id="row.item.id" />
                     </b-card>
                 </template>
-                <template v-slot:cell(details)="data">
-                    <b-button
-                        v-b-tooltip.hover.bottom
-                        title="Show Invocation Details"
-                        class="btn-sm fa fa-chevron-down"
-                        v-if="!data.detailsShowing"
-                        @click.stop="swapRowDetails(data)"
-                    />
-                    <b-button
-                        v-b-tooltip.hover.bottom
-                        title="Hide Invocation Details"
-                        class="btn-sm fa fa-chevron-up"
-                        v-if="data.detailsShowing"
-                        @click.stop="swapRowDetails(data)"
-                    />
-                </template>
                 <template v-slot:cell(workflow_id)="data">
-                    <div v-if="!ownerGrid || !getWorkflowByInstanceId(data.item.workflow_id)">
-                        {{ data.item.workflow_id }}
-                    </div>
-                    <div v-else>
-                        <workflow-dropdown :workflow="getWorkflowByInstanceId(data.item.workflow_id)" />
-                    </div>
+                    <b-link v-b-tooltip title="Show Invocation details" href="#" @click.stop="swapRowDetails(data)">
+                        <b>{{ getWorkflowNameByInstanceId(data.item.workflow_id) }}</b>
+                    </b-link>
                 </template>
                 <template v-slot:cell(history_id)="data">
-                    <div v-if="!ownerGrid || !getHistoryById(data.item.history_id)">
-                        {{ data.item.history_id }}
-                    </div>
-                    <div v-else>
-                        <history-dropdown :history="getHistoryById(data.item.history_id)" />
-                    </div>
+                    <b-link
+                        v-b-tooltip
+                        title="Switch to History"
+                        href="#"
+                        @click.stop="switchHistory(data.item.history_id)"
+                    >
+                        <b>{{ getHistoryNameById(data.item.history_id) }}</b>
+                    </b-link>
+                </template>
+                <template v-slot:cell(create_time)="data">
+                    <UtcDate :date="data.value" mode="elapsed" />
+                </template>
+                <template v-slot:cell(execute)="data">
+                    <b-button
+                        v-b-tooltip.hover.bottom
+                        title="Rerun Workflow"
```

Comment:
```suggestion
                        title="Run Workflow"
```
We will want a re-run button in the future, but that will need some backend work to figure out the parameters.

Reply:
Thank you for your suggestions. I relabeled the button to "Run", added the update time and the workflow invocation id.